app/task_routes.py

```diff
@@ -0,0 +1,131 @@
+from app import db
+from app.models.task import Task
+from flask import Blueprint, jsonify, abort, make_response, request
+import datetime
+import requests
+from app import token 
+
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+def validate_task(id):
+    try:
+        id = int(id)
+    except:
+        abort(make_response({"message": f"Task {id} is invalid"}, 400))
+
+    task = Task.query.get(id)
+
+    if not task:
+        abort(make_response({"message": f"Task {id} not found"}, 404))
+
+    return task
+
+@tasks_bp.route("", methods=["POST"])
+def create_task():
+    if request.method == "POST":
+        request_body = request.get_json()
+        if "title" not in request_body or "description" not in request_body:
+            return make_response(jsonify({"details": "Invalid data"}), 400)
```

Comment:
Nice job adding error handling in case the data in request_body that the client sent over is invalid 👍 

Another way to add error handling to the route would be to use try/except, like:
```python
try: 
    new_task = Task.from_dict(request_body) 
    db.session.add(new_task) 
    db.session.commit() 
    task_dict = dict(task=new_task.to_dict())

    return make_response(jsonify(task_dict), 201)
except KeyError: 
    abort(make_response({"details" : "Invalid Data"}))